platform/src/main/java/org/stellar/anchor/platform/data/CustodyTransactionStatus.java

```diff
@@ -0,0 +1,33 @@
+package org.stellar.anchor.platform.data;
+
+import com.google.gson.annotations.SerializedName;
+
+public enum CustodyTransactionStatus {
+  @SerializedName("created")
+  CREATED("created"),
+  @SerializedName("sent")
```

Comment:
The names (`sent` and `submitted`) can be consistent ?

Reply:
Fixed in #1067